.github/workflows/test-application.yaml

```diff
@@ -263,6 +264,7 @@ jobs:
             MAILER_URL: null://localhost
             SULU_ADMIN_EMAIL:
             COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            COMPOSER_ROOT_VERSION: dev-feature/content-resolver # todo remove when content bundle is merged into sulu/sulu
```

Comment:
this should be reverted when https://github.com/sulu/SuluContentBundle/pull/271 merged or content bundle is sulu/sulu